.github/workflows/integration-linux.yml

```diff
@@ -292,6 +292,270 @@ jobs:
         if: steps.deploy.outcome != 'success'
         run: kubectl get po
 
+  go-helm-dry-run:
+      runs-on: ubuntu-latest
+      needs: build
+      steps:
+        - uses: actions/checkout@v3
+        - uses: actions/download-artifact@v3
+          with:
+            name: draft-binary
+        - run: chmod +x ./draft
```

Comment:
💡 Note: `+x` for all? as default could be too much between user, group and others, just an fyi. 

Recommendation: `755` could be better but I am not sure what context will this run. hence sharing my thoughts.